core/build.rs

```diff
@@ -2,10 +2,10 @@ extern crate rustc_version;
 use rustc_version::{version, Version};
 
 fn main() {
-    if version().unwrap() >= Version::parse("1.25.0").unwrap() {
+    if version().unwrap() >= Version::parse("1.25.0").unwrap() || cfg!(feature = "i-am-libstd") {
         println!("cargo:rustc-cfg=has_repr_align");
     }
-    if version().unwrap() >= Version::parse("1.26.0").unwrap() {
```

Comment:
The build script won't be executed when the crate is built as a submodule.

Reply:
Ah, right. Was so much nicer to not need the `cfg(feature = "i-am-libstd")` everywhere in this code. But I guess I have to go back to that to get these features.